### ISSUE TITLE
Allow Dynamic Files to update more often

### ISSUE THREAD

**Issue by drspaceboo** [JimmyAppelt/Snaz#155]
Would it be possible to lower the delay between updates for Dynamic files that hit a server? I'm using it to display the name of the currently playing song, and having that update every 20 seconds isn't great. I've had some viewers call out it looks "broken" on my stream.

**Comment by JimmyAppelt** [comment]
I could yes, it is actually a little precaution. You can imagine when someone is able to set it to 1 second and hit a server with a very big list.

I could however build in a secret command to disable the restriction, I'l have to think about it.

**Comment by drspaceboo** [comment]
Yeah I figured there would be something about queuing of requests or something as to why you didn't allow it.

I would really love that secret command if that is quick and easy to put in.

I would say having settings for individual files, and queuing commands with a max number of connections at once or something would be the best ultimate solution.

**Comment by chriskewl** [comment]
I have my own private website and I'd hit it every second, especially if it's local. I would like that to happen.
